docs/how-to-get-help.md

```diff
@@ -1,15 +1,65 @@
-## Onboarders
+## Co-pilot programme
 
-When your project has been approved by NHS England, we will agree an intensive support period with you when our experienced researchers and engineers will be made available to get you started.
+When your project has been approved by NHS England,
+we will agree with you an intensive programme of support with our experienced researchers, or *co-pilots*.
```

Comment:
Would it be better to say that "we *can* agree with you an intensive programme of support"?

Reply:
Whilst I have added line breaks, I haven't changed the text: previously, we said "will". Is the co-pilot programme optional? When would we not arrange a co-pilot?

Reply:
Maybe new members of the team? Non-Bennett people who have previously been copiloted?

Reply:
Sounds reasonable 🙂 Updated by bb906a4.